addons/calendar/static/src/scss/calendar_event_views.scss

```diff
@@ -0,0 +1,16 @@
+.modal-dialog:has(.calendar_event_form_quick_create) {
+    --modal-width: 450px;
```

Comment:
why not just width: 450px ?

Reply:
Actually as long as there is some selector specificity, which is the case here, you may do as you prefer :)